src/App.jsx

```diff
@@ -1,33 +1,34 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { useEffect, useState } from 'react'
+import { Routes, Route } from 'react-router'
+// import reactLogo from './assets/react.svg'
+// import viteLogo from '/vite.svg'
 import './App.css'
+import Header from './Components/Header'
+import Footer from './Components/Footer'
+import Nav from './Components/Nav'
+import ArticleList from './Components/ArticleList'
+import { getTopics } from './Components/Api'
+
 
 function App() {
-  const [count, setCount] = useState(0)
+
+  const [topics, setTopics] = useState([])
+
+  useEffect(() => {
+    getTopics().then((topics) => {
+      setTopics(topics)
+    })
+  }, [])
+  
```

Comment:
This isn't part of the ticket but if you are going to fetch topics it could be done in your `ArticleList` as that is currently the only component that is using this data